hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -503,9 +519,16 @@ public void setPathProperties(final Path path,
       } catch (CharacterCodingException ex) {
         throw new InvalidAbfsRestOperationException(ex);
       }
+      final String relativePath = getRelativePath(path);
+      final EncryptionAdapter encryptionAdapter
+          = createEncryptionAdapterFromServerStoreContext(relativePath,
+          tracingContext);
       final AbfsRestOperation op = client
           .setPathProperties(getRelativePath(path), commaSeparatedProperties,
-              tracingContext);
+              tracingContext, encryptionAdapter);
```

Comment:
This is in-line with https://github.com/apache/hadoop/pull/3440#pullrequestreview-1133031625, where we don't want AbfsClient to have logic to get an encryptionAdapter: https://github.com/sumangala-patki/hadoop/blob/HADOOP-17912/hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java#L251-L262